### PR TITLE
Create landpage

### PR DESCRIPTION
**Pull request by NicolasAndrehh**
# Milestone - Landpage with JS interactivity
This pull request includes the following changes:
- Add HTML structure without styles
- Add Books array
- Add function to add a book to the array 
- Add function to remove a book
- Add localStorage to store Books array and input values

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  const removeBookButton = document.querySelectorAll('.show-books-section button');
+  removeBookButton.forEach((button) => {
+    button.addEventListener('click', () => {
+      /* eslint-disable no-use-before-define */
+      removeBook(button.className);
+    });
+  });
+}
+showBooks();
```
Comment (mckent05) [inline main.js]: - To make your project clean and look more professional, I suggest that you attach and call this function using an eventlistener, in this case I suggest using `DOMContentLoaded`. Please this 🙏

File: main.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  {
+    id: Date.now().toString(36) + Math.random().toString(36).substr(2, 3),
+    title: 'Black Clover',
+    author: 'Yuki Tabata',
+  },
+];
+
+if (localStorage.getItem('books')) {
+  bookList = JSON.parse(localStorage.getItem('books'));
+}
```
Comment (mckent05) [inline main.js:11]: - According to the project requirements, the bookList is used to originally save the list of books, I do not think there is any need to reassign the value of the bookList. Please fix this 👍 👍

File: main.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                <h1>Awesome books</h1>
+                ${bookRow}
+            `;
+    });
+  }
+
+  const removeBookButton = document.querySelectorAll('.show-books-section button');
+  removeBookButton.forEach((button) => {
+    button.addEventListener('click', () => {
+      /* eslint-disable no-use-before-define */
```
Comment (mckent05) [inline main.js]: - Please remove all disabled linters, rememeber that linters help us ensure that we write code that is clean and easy to maintain. Please fix this 🙏

File: index.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+                <input type="text" class="title-input" placeholder="Title" required>
+                <input type="text" class="author-input" placeholder="Author" required>
+                <input type="submit" id="add-book-button">
+            </form>
+        </section>
+    </main>
+
+    <script src="main.js"></script>
+</body>
+</html>
```
Comment (mckent05) [inline index.html:26]: - I get this error when I launch your project. Please fix this issue for a better UI/UX. 🙏 
wrwrgrg

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @NicolasAndrehh, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NicolasAndrehh** [comment]
# Hi @mckent05!
Thank you for your feedback we're going to do the changes that you suggested!

File: main.js (hunk 2 above)
Comment (NicolasAndrehh) [reply main.js:11]: That part is to use the localStorage array, I don't think I should change it @mckent05

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @NicolasAndrehh , 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**